suite-native/module-accounts-import/src/screens/XpubScanScreen.tsx

```diff
@@ -44,6 +44,12 @@ const cameraStyle = prepareNativeStyle(utils => ({
     marginBottom: utils.spacings.medium,
 }));
 
+const isBtcTestnetXpub = (xpubAddress: string) => {
+    const btcTestnetPrefixes = ['tpub', 'upub', 'vpub', 'Upub', 'Vpub'];
+
+    return btcTestnetPrefixes.some(prefix => prefix === xpubAddress.slice(0, 4));
```

Comment:
This won't work with Taproot xpubs, because of the taproot prefix. What about removing the taproot prefix first if it is present in the `xpubAddress` string?

But anyway even if you fix this, there are other unhandled cases, as you said today during the standup. So not sure how much time we want to invest to this.

Reply:
Good point. This will fix that.
https://github.com/trezor/trezor-suite/pull/11329/commits/9c5f5c9331db0e6faa0acad25105a0bb2583ab1d